README_LINUX.md

```diff
@@ -108,7 +108,28 @@ Unfortunately, the Qt installer does not allow you to deselect the multi-gigabyt
 Building
 --------
 
-### Step 1: Make a build directory
+### Step 1: Obtain the source code
+
+SuperCollider is hosted on Github: https://github.com/SuperCollider/SuperCollider
+
+Obtaining the SuperCollider source code can be done either by downloading a release tarball, or by cloning the repository.
+
+SuperCollider releases are available to download here: https://github.com/supercollider/supercollider/releases
+
+Cloning the repository can be done with the following command:
+
+    git clone --recurse-submodules https://github.com/SuperCollider/SuperCollider.git
+
+The `--recurse-submodules` option will clone the repository's submodules which are needed to build SuperCollider. If you forget to clone them, you'll get error messages like the one listed below later on in the build process:
+
+  [SUBMODULE-NAME] is missing: please run `git submodule init && git
+  submodule update' from the toplevel of your git working tree
+
+The submodules can be cloned manually by navigating to the root your locally cloned SuperCollider repository and running the following command:
+
+  git submodule update --init
```

Comment:
I don't think any of this (from "If you forget to clone them" onward) needs to be restated; it's obvious from the error message. If you don't think it's clear enough, update that warning text in the cmake file.

Reply:
Thanks for reviewing @brianlheim. I removed the part mentioning the cmake error message. I like it much better now. @snappizz asked that I add this stuff during the last dev meeting.

Reply:
yeah that nathan guy's a damn fool, don't listen to anything he says

Reply:
I realize that my comment came out harsher than I intended it to @snappizz. I simply wanted to let @brianlheim know that we discussed it during the last meeting. You ain't no damn fool :)

In any case, this bit of text is better now than it was. Thanks for the reviews!